examples/main.cpp

```diff
@@ -45,7 +45,8 @@ int bc::main(int argc, char* argv[])
 #endif
 
     // Extracting Satoshi's words from genesis block.
-    const auto block = bc::chain::block::genesis_mainnet();
+    const auto block =
+        bc::chain::block::genesis_mainnet(libbitcoin::settings());
```

Comment:
use `bc::settings()`

Reply:
The genesis block needs to itself become a setting, meaning that `genesis_mainnet` will no longer exist. The genesis block will be deserialized from the base16-encoded setting stream.

Reply:
Yes, the genesis block is on my todo list. I prefer to approach the migration of the constants iteratively. Once this is merged I'd open a new PR where the genesis block becomes a setting.

Reply:
done